.github/workflows/test-e2e.yml

```diff
@@ -117,7 +117,7 @@ jobs:
     uses: broadinstitute/sherlock/.github/workflows/client-report-workflow.yaml@main
     if: ${{ github.ref == 'refs/heads/develop' }}
     with:
-      notify-slack-channels-upon-workflow-completion: '#jade-alerts,#dsde-qa'
+      notify-slack-channels-upon-workflow-completion: '#jade-spam'
```

Comment:
I could see sending all alerts to #jade-spam, but would we also like to send alerts on workflow failure to #jade-alerts still, since a failure is something that we'd likely want to look at even if not officially a release blocker?

Reply:
I have the same question- why remove from jade-alerts?

Reply:
I would prefer to have both of the UI test outputs in the same spot and there was pressure to move things out of jade-alerts. I personally check both channels. 

I definitely am not tied to this setup, but I think I will push ahead with this change.  If we find that the results are getting ignored or we want to change it for any reason, I'm glad to put in another PR!